content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx

```diff
@@ -7,39 +7,59 @@ preview: true
 
 # Full-Text Search (Preview)
 
-The Prisma Client supports a full-text search API for Postgres databases. With full-text search enabled, you can add search functionality to your application by searching for text within a database column.
+The Prisma Client supports full-text search API for **PostgreSQL** and **MySQL** databases. With full-text search enabled, you can add search functionality to your application by searching for text within a database column.
```

Comment:
Should we include version information here? MySQL from 3.8.0, Postgres from when?

Reply:
PostgreSQL = https://github.com/prisma/prisma/releases/tag/2.30.0 (via https://github.com/prisma/prisma/issues/8877)

Reply:
Done